### ISSUE TITLE
.COM expiry date

### ISSUE THREAD

**Issue by hlestrat** [gen1us2k/python-whois#42]
https://github.com/gen1us2k/python-whois/blob/21693a468e4e2262ba5937b41e33a45e2da98330/whois/tld_regexpr.py#L10

Hello,
Some ".com" registrar (Gandi SAS at least) do not only give the expiration date as "Expiration Date", but "Registry Expiry Date".

So I propose to change the regex from 
'expiration_date': r'Expiration Date:\s?(.+)' to
'expiration_date': r'Expir.* Date:\s?(.+Z)' (well, Z is not obvious to be necessary)

Regards

**Comment by tierpod** [comment]
Hello. Could you give example of such domains? I'm planning to write test and fix for it.

**Comment by hlestrat** [comment]
Hello,

Well, I do not remember exactly, but I just checked one domain, and it seems to return two "expirations"
whois boursorama.com | grep -i expir
Registry Expiry Date: 2021-02-28T05:00:00Z
...
Registrar Registration Expiration Date: 2021-02-28T00:00:00+01:00Z

Actually, I do not remember the exact problem. All I remember is that I did put in your code my fix proposal and that solved my problem at that time. Sorry to not be more specific.
